docs/overview.md

```diff
@@ -882,4 +882,37 @@ Dendrograms are tree-like driagrams that represent hierarchical relationships an
+
+# Appendix
+
+## Patristic and Cophenetic definitions
+
+The below image provides an overview of the difference between **patristic distance** and **cophenetic distance** used by this software for the produced trees from hierarchical clustering.
+
+![patristic-cophenetic.png][]
+
+### Patristic
+
+A definition of **patristic distance** can be found in *Fourment, M., Gibbs, M.J. PATRISTIC: a program for calculating patristic distances and graphically comparing the components of genetic change. BMC Evol Biol 6, 1 (2006). https://doi.org/10.1186/1471-2148-6-1*.
+
+> A patristic distance is the sum of the lengths of the branches that link two nodes in a tree, where those nodes are typically terminal nodes that represent extant gene sequences or species.
+
+## Cophenetic
+
+The cophenetic distance of two objects (samples or hierarchical clusters) is a measure of how similar those objects need to be in order to be grouped into the same cluster. This distance is the height of the dendrogram at which point two objects are hierarchically clustered together. See the [Wikipedia article on Cophenetic][wiki-cophenetic].
+
+However, in [scikit-bio][scikit-bio-cophenet]'s documentation, the cophenetic distance has a different definition (used for the `cophenet` function):
+
+>The cophenetic distance [1] between a pair of tips is essentially the sum of branch lengths connecting them (i.e., patristic distance [2], see distance). It measures the divergence between two taxa in evolution.
+
+However, note they also acknowledge the other definition of cophenetic distance that we are using:
+
+>In hierarchical clustering, the cophenetic distance is commonly used to measure the dissimilarity between two objects before they are joined in a dendrogram. In that context, it is also defined as the height of the lowest common ancestor (LCA) from the surface of the tree. However, phylogenetic trees are usually non-ultrametric (e.g., nj), and the two child clades of a node may have different heights. Therefore, the cophenetic distance is instead defined as the patristic distance between the two tips. For ultrametric trees (e.g., upgma), this method’s result should match SciPy’s cophenet.
+
+We use this definition (height of lowest common ancestor) since this code is used for hierarchical clustering and produces ultrametric trees.
```

Comment:
I would argue we use the definition "the distance at which they cluster", which in turn is being interpreted as a ancestor. I don't think it's a bad assumption to make, but it's one step from cophenetic (cophenetic -> distance at which they cluster -> LCA).